Core/Core/Planner/CalendarEvent/ViewModel/CalendarEventDetailsViewModel.swift

```diff
@@ -74,17 +74,17 @@ public final class CalendarEventDetailsViewModel: ObservableObject {
         self.completion = completion
         self.deleteConfirmation = deleteSingleItemConfirmation
 
-        onEventSetUpdateDeleteConfirmationModel()
-        onEventSetUpdateMenuButtonVisibility(userId: userId)
+        updateDeleteConfirmationModel(onSet: event)
+        updateMenuButtonVisibility(onSet: event, userId: userId)
 
         loadData()
 
-        onEditButtonTapShowEditScreen()
-        onDeleteButtonTapDeleteEventAfterConfirmation()
+        showEditScreen(on: didTapEdit)
+        deleteEventAfterConfirmation(on: didTapDelete)
     }
 
-    private func onEventSetUpdateDeleteConfirmationModel() {
-        event
+    private func updateDeleteConfirmationModel(onSet subject: CurrentValueSubject<CalendarEvent?, Never>) {
```

Comment:
Here as well, can we rename argument name to be `on` rather `onSet`. 
Or, `onSubject` for more expressive one.

Reply:
Same as above